src/test/java/org/gridsuite/geodata/server/GeoDataServiceTest.java

```diff
@@ -837,4 +845,136 @@ private Network createCgmesGeoDataNetwork() {
 
         return network;
     }
+
+    @Test
+    public void testGetGeodataById() {
```

Comment:
Separe tests to getSubstations geoData by Id and another one to getLines geoData by Id ?

Reply:
done